lib/rubocop/cop/naming/private_method_name.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Naming
+      # Checks that private methods names
+      # do not start without a underscore prefix.
```

Comment:
Might be good to mention here that the convention is used to some extent (e.g. in Rails), but it's not widely adopted in the community and that's the reason why this cop is disabled by default.